cfme/tests/cloud_infra_common/test_cockpit_server.py

```diff
@@ -42,14 +40,7 @@ def test_cockpit_server_role(appliance, provider, setup_provider, new_vm, enable
 
     view = navigate_to(new_vm, 'Details')
```

Comment:
**REQUIRED** I believe the conditional can be replaced with this
```python
    view = navigate_to(new_vm, 'Details')
    assert view.toolbar.access.item_enabled('Web Console') == enable
```

Reply:
@psav It's a parametrized test with checking both `True` and `False` condition for cockpit, means if its not enabled, check its actually off, and on if its actually enabled.

Reply:
@niyazRedhat the assertion psav is suggesting takes that into account - here you've just asserted the boolean returned by item_enabled within if blocks controlled by `enable`, instead of asserting directly against that boolean compared to `enable`.  Functionally its the same, psav's suggestion is just simpler overall syntax.

Since you take an action when `enable` is `True` anyway, that's not taken when false, the if block will be required anyway, and I don't see any harm in asserting against the boolean returned by `item_enabled`.